pages/research_platforms.md

```diff
@@ -3,16 +3,296 @@ title: Research Platforms
 permalink: /platforms/
 breadcrumb: true
 ---
-
+Welcome to the University of Leeds' Research Computing Resource Guide!
 IT Services provide a number of tools and services for both data storage and compute, Research Computing being the primary point of contact for research users.
 
+## Finding the Right Computing Resources for Your Research
+
+Whether you're working on small-scale analysis, developing complex simulations, or running intensive computations, we have a wide range of computing resources tailored to support your research.
+
+This guide will help you navigate through various options, from individual devices to advanced High-Performance Computing (HPC) systems. To find the most appropriate resource, consider these questions:
+
+- Are you looking for something you can use immediately on your personal device?
+- Do you need more power without the complexity of HPC systems?
+- Are you running large-scale computations that require multi-node parallel processing?
+
+*Use our Resource Roadmap below to discover the best solution for your research needs.*
+
+### Resource Roadmap
+
+1. **Starting Small: Individual Device**
+   - If you're just getting started or your computations are lightweight, you can easily begin on your laptop or desktop.
+   - Ideal for initial code development, small data analysis, and non-intensive simulations.
+2. **Need More Power? Try Shared Devices**
+   - If your work is starting to slow down your individual device, consider using a cluster machine or Linux server.
+   - These shared systems give you extra memory, CPU power, or GPU access without the complexity of an HPC queue. Great for testing and debugging.
+3. **Cloud Solutions for Flexibility and Collaboration**
+   - For collaborative projects, cloud resources offer flexible environments.
+   - These are perfect for sharing work, developing in teams, and quickly testing your code without investing in hardware.
+   - Unfortunately, most of the solutions are not supported by the university
+4. **Need to Scale Up? Local HPC is the Next Step**
+   - When your research requires heavy computation, high-end GPUs, or parallel processing, it’s time to consider our local (Tier 3) HPC resources like.
+   - Get direct support from the Research Computing team and access systems tailored to your project’s needs.
+5. **For the Largest Projects: External HPC**
+   - For large, multi-node, or specialized GPU jobs, Tier-2 (Regional) and Tier-1 (National) HPC facilities offer matchless power and scale.
```

Comment:
```suggestion
   - For large, multi-node, or specialized GPU jobs, Tier-2 (Regional) and Tier-1 (National) HPC facilities offer unmatched power and scale.
```

Reply:
Also, you put "Tier 3" higher up but "Tier-2" here.  Pick on or the other style.